docs/guide/adapters/README.md

```diff
@@ -346,6 +357,21 @@ _(in order of first appearance)_
   
     ![](../../images/LP-CC1352P7.jpg)
     </details>
+* JetHub D1+
```

Comment:
```suggestion
* Hub
```

Reply:
but
`* Wiren Board`

I added `Hub` at the beginning. Or would it be better to separate Wiren Board/JetHub into a separate Hub category?

Reply:
Yes, let's put them into a separate category called `Hub`

Reply:
Fixed